timescaledb/tutorials/analyze-intraday-stocks/fetch-and-ingest.md

```diff
@@ -0,0 +1,230 @@
+# 2. Fetch and insert intraday stock data
+
+In this step, you will
+
+* create a configuration file (optional)
+* fetch stock data
+* insert the data into TimescaleDB
+
+## Create a configuration file
+
+This is an optional step, but it is highly recommended not to store your password or other sensitive information
+directly in your code. Instead, create a configuration file, for example `config.py`, and include your 
+database connection details and Alpha Vantage API key in there:
+
+```python
+# example content of config.py:
+DB_USER = 'user'
+DB_PASS = 'passwd'
+DB_HOST = 'host'
+DB_PORT = '000'
+DB_NAME = 'db'
+APIKEY = 'alpha_vantage_apikey'
+```
+
+Later, whenever you need to reference any of the information from this config file, you just need to import it:
+```python
+import config
+apikey = config.APIKEY
+...
+```
+
+## Collect ticker symbols
+
+In order to fetch intraday stock data, you will need to know which ticker symbols you want to analyze.
+
+First, let's collect a list of symbols so later we can fetch their data!
+
+In general, you have a few options to gather a list of ticker symbols dynamically:
```

Comment:
```suggestion
In order to fetch intraday stock data, you will need to know which ticker symbols you want to analyze.
First, let's collect a list of symbols so that we can fetch their data later.
In general, you have a few options to gather a list of ticker symbols dynamically:
```